lib/sinatra/soap/helper_methods.rb

```diff
@@ -9,15 +9,84 @@ def soap_views()
         File.join(File.dirname(__FILE__), "..", "views")
       end
 
+      def hash_to_xml(xml, hash)
```

Comment:
Yep. But what you think about this idea in general? (`@..` for xml attributes, `@@content` to create tag with attributes and simple content, converting array list of singularized tags)